packages/ui-components/src/components/Map/Markers/markerColors.js

```diff
@@ -0,0 +1,223 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+
+import moment from 'moment';
+import { blue, red, green, lightGreen, yellow, orange } from '@material-ui/core/colors';
+
+export const BREWER_PALETTE = {
+  yellow: '#F0F032',
+  navy: '#005AC8',
+  purple: '#8214A0',
+  teal: '#006E82',
+  custard: '#FAE6BE',
+  blue: '#00A0FA',
+  orange: '#FA7850',
+  magenta: '#FA78FA',
+  cyan: '#14D2DC',
+  green: '#0AB45A',
+  red: '#AA0A3C',
+  lime: '#A0FA82',
+};
+
+export const TUPAIA_ORANGE = '#EE6230';
```

Comment:
There is a card in the backlog to update the colors to be derived from a material-ui theme.https://github.com/beyondessential/tupaia-backlog/issues/2213